test/controllers/api/v2/override_values_controller_test.rb

```diff
@@ -128,4 +128,46 @@ class Api::V2::OverrideValuesControllerTest < ActionController::TestCase
     end
     assert_response :error
   end
+
+  test "should not accept boolean that is not sent as string" do
```

Comment:
Is there a test for accepting booleans contained in strings, e.g: testing that I pass `"true"` and the overridden value is actually saved as a boolean? It works but I had to test it manually.